src/darwin/darwin-host-session.vala

```diff
@@ -572,6 +576,14 @@ namespace Frida {
 			handle_spawn.begin (info);
 		}
 
+		private void on_resumed (uint pid) {
+			launchd_agent.forget_pid.begin (pid, io_cancellable);
+		}
+
+		private void on_killed (uint pid) {
+			launchd_agent.forget_pid.begin (pid, io_cancellable);
+		}

```

Comment:
Perhaps these would read slightly better as `on_process_resumed` and `on_process_killed`?